casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -155,16 +155,16 @@ sealed abstract class MultiParentCasperInstances {
                     else if (!validSender) InvalidUnslashableBlock.pure[F]
                     else attemptAdd(b)
           _ <- attempt match {
-                case MissingBlocks         => ().pure[F]
```

Comment:
You don't need to change either of these lines because the return type of `reAttemptBuffer` is `F[Unit]`